src/interpolations.jl

```diff
@@ -363,6 +367,15 @@ boundarydof_indices(::Type{FaceIndex}) = Ferrite.facedof_indices
 boundarydof_indices(::Type{EdgeIndex}) = Ferrite.edgedof_indices
 boundarydof_indices(::Type{VertexIndex}) = Ferrite.vertexdof_indices
 
+"""
+    is_discontinuous(::Interpolation)
+    is_discontinuous(::Type{<:Interpolation})
+
+Checks whether the interpolation is discontinuous (i.e. `DiscontinuousLagrange`)
+"""
+is_discontinuous(::Interpolation) = false
```

Comment:
Can canonicalize the answers like so:
```suggestion
is_discontinuous(ip::Interpolation) = is_discontinuous(typeof(ip))
```